action.yml

```diff
@@ -14,14 +14,17 @@ inputs:
 outputs:
   release_version:
     description: 'The release version of Heroku deployment, e.g. 10'
+    value: ${{ steps.deploy-and-get-release-version.outputs.release_version }}
 
 runs:
   using: "composite"
   steps:
-    - run: python3 -m deploy_heroku
+    - id: deploy-and-get-release-version
+      run: python3 -m deploy_heroku
       shell: bash
       env:
         APP: ${{ inputs.heroku_app }}
         API_KEY: ${{ inputs.heroku_api_key }}
         COMMIT_HASH: ${{ inputs.github_sha }}
+        EVENT_PAYLOAD: ${{ toJson(github.event.deployment.payload) }}
```

Comment:
Ich frage mich was passiert, wenn in payload nichts drin ist - ich gehe aber davon aus (laut tests), dass es ein leeres dict kommt. Im übrigens, danke für die Tests ❤️

Reply:
Das war leider nicht korrekt. Das ist ein String, wenn der Payload nicht gesetzt ist, ist es ein Leerstring.
Habe ich angepasst.